frontend/src/views/Home.js

```diff
@@ -1,12 +1,18 @@
 import React from 'react';
-import Button from "../components/Button/Button";
+import Button from '../components/Button/Button';
 
-function Home() {
-  return (
-    <div className="home">
-      <Button to="/company" label="Register your business" type="primary"/>
+const Home = () => (
+  <div className="Home">
+    <div className="Home-hero">
+      <h1 className="Home-heading">Two line describtion of the idea and service.</h1>
+      <p className="Home-text">Just a tiny bit more explanation, since two lines probably won’t tell enough.</p>
     </div>
-  );
-}
 
-export default Home;
+    <div className="Home-btns">
+      <Button to="/map" label="Search for shops nearby!" />
+      <Button to="/company" label="Register your business" secondary />
+    </div>
```

Comment:
We should have a button grp component for that, not something you need to add now before merging but just so you know that you can still add this later on.